### DISCUSSION TITLE
Grafana 11 + Omniperf plugin + Angular JS

### DISCUSSION THREAD

**Discussion by ELCapitanLLNL** [AMDResearch/omniperf#415] [Q&A] [answered]
Hello! 
I am wondering if anyone experiencing an issue with the latest Grafana version (11) and Omniperf plug-in. 
I am getting: `error msg="Refusing to initialize plugin because it's using Angular, which has been disabled" pluginId=amd-omniperf-data-plugin`  
info: https://grafana.com/blog/2024/03/11/removal-of-angularjs-support-in-grafana-what-you-need-to-know/
Thanks!

**Comment by coleramos425** [comment]
Thank you for sharing @ELCapitanLLNL. I was able to reproduce your issue and filed a ticket to track (#416). The short-term solution is to rollback Grafana version to enable our AngularJS plugin. Meanwhile, we'll assign an engineer to port the plugin to React .

I'll reach back out in this thread when the patched version is available.

**Reply by ELCapitanLLNL** [reply]
Hello @coleramos425 ,
We are wondering if there is any progress regarding to migrate the plug-in to React, giving that Grafana 10.4 EOL is in June 2025.
I tried to enable angularJS in Grafana 11 via **angular_support_enabled=true** configuration parameter but no luck.
Thanks,
ElCap

**Comment by coleramos425** [comment, marked as answer]
Patch #417 has been upstreamed to the staging branch `amd-staging`. You can checkout that branch to preview the fix ahead of the next Omniperf release

**Reply by ELCapitanLLNL** [reply]
Thank you Ramos, I can confirm the patch works on my end as well.  I am wondering if you are planning to push the Grafana  image into Dockerhub : https://hub.docker.com/u/colramos  
Thanks!

**Reply by coleramos425** [reply]
We do not maintain an image on DockerHub for Omniperf's Grafana integration. If that is something your lab would find useful I'd encourage you to file a [feature request](https://github.com/ROCm/omniperf/issues/new/choose) and we can make it happen. Thanks!

**Reply by ELCapitanLLNL** [reply]
Done! https://github.com/ROCm/omniperf/issues/418
Thank you Mr. Ramos.
